templates/repo/issue/list.tmpl

```diff
@@ -2,6 +2,88 @@
 <div role="main" aria-label="{{.Title}}" class="page-content repository issue-list">
 	{{template "repo/header" .}}
 	<div class="ui container">
+
+	{{if .PinnedIssues}}
+		<div id="issue-pin-div" {{if .IsRepoAdmin}}data-is-repo-admin{{end}} style="display: flex; flex-grow: 1; flex-wrap: wrap;">
+			{{range .PinnedIssues}}
+				<div class="card pinned-issue-card" data-move-url="{{$.Link}}/pin_move" data-issue-id="{{.ID}}">
+					{{if eq $.Project.CardType 1}}
+						<div class="card-attachment-images">
+							{{range (index $.issuesAttachmentMap .ID)}}
+								<img src="{{.DownloadURL}}" alt="{{.Name}}">
+							{{end}}
+						</div>
+					{{end}}
+					<div class="content gt-p-0">
+						<div class="header">
+							<span class="gt-dif gt-ac gt-vm {{if .IsClosed}}red{{else}}green{{end}}">
+								{{if .IsPull}}
+									{{if .PullRequest.HasMerged}}
+										{{svg "octicon-git-merge" 16 "text purple"}}
+									{{else}}
+										{{if .IsClosed}}
+											{{svg "octicon-git-pull-request" 16 "text red"}}
+										{{else}}
+											{{svg "octicon-git-pull-request" 16 "text green"}}
+										{{end}}
+									{{end}}
+								{{else}}
+									{{if .IsClosed}}
+										{{svg "octicon-issue-closed" 16 "text red"}}
+									{{else}}
+										{{svg "octicon-issue-opened" 16 "text green"}}
+									{{end}}
+								{{end}}
```

Comment:
I think there is a shared template for these icons somewhere, if not, make one where possible.

Reply:
Why do we need a template for the octicon? It's just one line.

Reply:
I'm talking about the whole block above, which is like 30 lines and is definetly something that should not be repeated across the codebase.

Reply:
The code is exactly the same as in `templates/shared/issuelist.tmpl`, so I suggest you create `templates/shared/issueicon.tmpl`, move the code there and then import is via `{{template}}` in both places.